tests/unit_tests/test_qmpc.py

```diff
@@ -154,6 +155,32 @@ def test_predict_errorhandring(self, run_server1,
                 model_param_job_uuid, model_id,
                 [["id1", "id2"], [3], [1]], [0, 1])
 
+    def test_exception_job_error(self):
+        # QMPCJobErrorとして例外がthrowされるかのテスト
+        qmpc: QMPCServer = QMPCServer(
+            ["http://localhost:9001",
+             "http://localhost:9002",
+             "http://localhost:9003"],
+            "QMPCJobError"
+        )
+        with pytest.raises(QMPCJobError):
+            qmpc.execute_computation(
+                1,
+                [["id1", "id2"], [0], [1, 1]], [[0, 1], []])
```

Comment:
If the mock sever side is modified, the `qmpc` declaration is no longer needed, so please remove it.
This makes the test easier to understand because the call corresponds to an error.
test_exception_server_error is too.

Reply:
Thanks for the review!
I fixed it with [bfeb25d](https://github.com/acompany-develop/QuickMPC-libClient-py/pull/24/commits/bfeb25d9975df0da1a4c1e6ecc59de8cc414402f).